### ISSUE TITLE
When is an update Coming?

### ISSUE THREAD

**Issue by GodsMasterpiece509** [the-dise/instander-android#21]

**Comment by illtellyoulater** [comment]
and hopefully it's coming with source code...?

**Comment by GodsMasterpiece509** [comment]
> and hopefully it's coming with source code...?

Hope so too

**Comment by theSoberSobber** [comment]
pretty sure never now, lol

**Comment by illtellyoulater** [comment]
yeah... I know, but if he doesn't upload it soon then it will start looking dodgy...  
To put the link to the binaries on here, benefiting from advertising it as a open source project, while there are no sources at all...

Despite I like the software very much I will be forced to report it if he won't upload the code in a week or two... as currently we have no idea what's in the code...

**Comment by theSoberSobber** [comment]
so start mass reporting ig?

**Comment by illtellyoulater** [comment]
@the-dise 

I'd be glad to avoid the above if possibile... 
Can you not just publish the code?
If you changed your mind and no longer want to release it as open source, then just upload the APK somewhere and post a link to that in [reddit.com/r/some_apk_subreddit](https://www.reddit.com/r/ApksApps/) ?
I don't think it's going to take you more than a week for this... :) Holding until next monday ...

**Comment by illtellyoulater** [comment]
Just noticed @the-dise is from Ukraine, or at least that's what it says on his profile https://github.com/the-dise  so he might be facing some slightly more important issue right now than this this one. 
I won't go on with the above reporting but until we know more I think the best option is uninstalling the app. 
Hope all is well with him.

**Comment by theSoberSobber** [comment]
understandable.

**Comment by GodsMasterpiece509** [comment]
> Just noticed @the-dise is from Ukraine, or at least that's what it says on his profile https://github.com/the-dise  so he might be facing some slightly more important issue right now than this this one. 
> I won't go on with the above reporting but until we know more I think the best option is uninstalling the app. 
> Hope all is well with him.

Yeah hope everything is well on his side if this is the case but atleast he could of done a public Communication to Allow us to know when or if one is coming i think Communication is key to Understand the Situation if he chooses to share if not what can we say..

**Comment by illtellyoulater** [comment]
Ok I think that rather than expecting updates to this repo we should at this point focus on finding alternative solutions. Are there any you are aware of? If so please share them here.
